immuni_common/models/mongoengine/temporary_exposure_key.py

```diff
@@ -47,6 +47,15 @@ def created_at(self) -> datetime:
             self.rolling_start_number * timedelta(minutes=10).total_seconds()
         )
 
+    @property
+    def expires_at(self) -> datetime:
+        """
+        Returns the datetime when the current key will expire
```

Comment:
_suggestion_: The other docstrings use `Return` instead of `Returns`.
In general, to keep consistency with the other docstrings, and since it's a property, I'd write the docstring as:
```python
"""
The datetime when the current key stops being used by the client to generate RPIs.
"""

Also, you're missing the definition in the `:return:`

Lastly, can we "safely" hardcode the 10 for now?

Reply:
Done ✅